user_docs/en/changes.md

```diff
@@ -88,7 +88,7 @@ There are many minor bug fixes for applications, such as Thunderbird, Adobe Read
 * Fixed a bug causing NVDA to fail to read the ribbon and options within Geekbench. (#16251, @mzanm)
 * Fixed a rare case when saving the configuration may fail to save all profiles. (#16343, @CyrilleB79)
 * In Firefox and Chromium-based browsers, NVDA will correctly enter focus mode when pressing enter when positioned within a presentational list (ul / ol) inside editable content. (#16325)
-* Column state change is automatically reported when selecting columns to display in Thunderbird message list. (#16323)
+* Column state change is now correctly reported when selecting columns to display in Thunderbird message list. (#16323)

```

Comment:
Nothing automatic here.

@burmancomp, do you agree with this change?

Or maybe even the following would be better in English (@seanbudd, your opinion?):
```suggestion
* The column state change is now correctly reported when selecting the columns to display in Thunderbird message list. (#16323)
```

Reply:
the original change you have proposed here is better